versioned_docs/version-2.0/pricing.mdx

```diff
@@ -69,7 +69,7 @@ trace step and again after it is complete)
 
 ### I’ve been using LangSmith since before pricing took effect for new users. When will pricing go into effect for my account?
 
-If you’ve been using LangSmith already, your usage will be billable starting sometime in May. At that point if you want to add seats or use more than the monthly allotment of free traces, you will need to add a credit card to LangSmith or contact sales. If you are interested in the Enterprise plan with higher rate limits and special deployment options, you can learn more or make a purchase by reaching out to [sales@langchain.dev](mailto:sales@langchain.dev).
+If you’ve been using LangSmith already, your usage will be billable starting sometime in June. At that point if you want to add seats or use more than the monthly allotment of free traces, you will need to add a credit card to LangSmith or contact sales. If you are interested in the Enterprise plan with higher rate limits and special deployment options, you can learn more or make a purchase by reaching out to [sales@langchain.dev](mailto:sales@langchain.dev).
```

Comment:
note: didn't fully revamp this page but added some content/made some small modifications